product/quickstart.mdx

```diff
@@ -1,62 +1,75 @@
-  </Card>
-  <Card title="SEO" icon="magnifying-glass-chart" href="/product/SEO">
-    Optimize SEO for your site more easily than ever
-  </Card>
-  <Card
-    title="Localization"
-    icon="map-location-dot"
-    href="/product/localization"
-  >
-    Manage content across different locales
-  </Card>
-  <Card
-    title="Publishing"
-    icon="rocket-launch"
-    iconType="solid"
-    href="/product/publishing"
-  >
-    Push changes to your live domain instantly
-  </Card>
-</CardGroup>
+If you already have a site and want to create a new one, click the **Create new site** button under the site list dropdown. If this is your first time using Makeswift, [create a free account](https://app.makeswift.com/signup). You'll then be taken to the page for creating a new site.
+
+You have two options to get started: **Start with the builder** or **Integrate with Next.js**.
+
+<Info>
+  Starting with the builder utilizes Makeswift's [default
+  host](/product/site/hosting#default-host) while Integrating with Next.js will
+  prompt you to utilize your own [custom
+  host](/product/site/hosting#custom-host).
+</Info>
+
+<Frame>![create a site](/images/quickstart/create-site.jpg)</Frame>
+
+Click **Go to the builder**, and you'll have a list of templates to choose from. Templates come with existing assets (images, logos, etc.), colors, text styles, and layouts for you to begin customizing.
+
+Choose the **Venture** template and click **Use this site**.
+
+<Frame>
+  <video
+    autoPlay
+    muted
+    loop
+    playsInline
+    title="Create new site from template"
+    className="w-full aspect-video"
+    src="/images/quickstart/create-site-from-template.mp4"
+  ></video>
+</Frame>
+
+## Visual Builder overview
+
+Once your new site is created, you'll be taken to the Visual Builder. Let's take a look at the different parts of the builder which is composed of 4 sections:
+
+<Tabs>
+  <Tab title="Navigation Sidebar">
+    Explore your sites, pages, files, and settings.
+    <Frame>
+    ![ Makeswift builder Navigation Sidebar](/images/quickstart/builder-layout-navigation-sidebar.jpg)
+    </Frame>
+  </Tab>
+
+<Tab title="Component Toolbar">
+  Access components to add to the Canvas.
+  <Frame>
+    ![ Makeswift builder
+    toolbar](/images/quickstart/builder-layout-component-toolbar.jpg)
+  </Frame>
+</Tab>
+
+<Tab title="Canvas">
+  Build your pages and see them come to life with an exact preview.
+  <Frame>
+    ![ Makeswift builder canvas](/images/quickstart/builder-layout-canvas.jpg)
+  </Frame>
+</Tab>
+
+  <Tab title="Properties Sidebar">
+    Edit properties of a page, site, or component.
+    <Frame>
+      ![ Makeswift builder panels](/images/quickstart/builder-layout-properties-sidebar.jpg)
+    </Frame>
+  </Tab>
+</Tabs>
+
+Before moving on, take a few minutes to explore the Canvas and its components. Click on any component in the Canvas to see its corresponding properties in the Properties Sidebar.
```

Comment:
From what I've chatted with people about, I think we were ok to just use "components" consistently instead of elements. That way, we don't have to distinguish the difference between a component vs an instance being an element for non-technical audiences. Any thoughts?

Reply:
I think that's probably fine. We might need more precise language if we're ever trying to compare two different instances of the same component, perhaps one we release visual component overrides. At that point we can decide how we want to distinguish these things (e.g., "component instances" vs "elements", etc.)